src/components/Navigation/LoginButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import ModalBUtton from '../Modal';
+
+
+
+export function LoginButton() {
```

Comment:
possible error in naming? the function is named `LoginButton` but the link points to register.